build.go

```diff
@@ -55,18 +56,21 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 		}
 
 		logger.Process("Determining start commands to include in procs.yml:")
+		watchPaths := make([]string, 0)
```

Comment:
Yeah. I've gotten into the habit of using `make` because it "just works" in any situation and don't always know if/when it will be used as per later test cases. If there's a preference the other way I'll use that convention.